state_machine/unit_tests/test_logs.py

```diff
@@ -54,15 +54,15 @@ def test(self):
 
         unpacked = unpack_beacon(pkt)
 
-        self.assertEqual(state_in, unpacked["state_index"])
-        self.assertEqual(f_contact_in, unpacked["contact_flag"])
-        self.assertEqual(f_burn_in, unpacked["burn_flag"])
-        self.assertEqual(error_count_in, unpacked["software_error_count"])
-        self.assertEqual(boot_count_in, unpacked["boot_count"])
-        self.assertAlmostEqual(vbatt_in + 0.01, unpacked["battery_voltage"], places=5)
-        self.assertAlmostEqual(cpu_temp_in, unpacked["cpu_temperature_C"], places=5)
-        self.assertAlmostEqual(imu_temp_in, unpacked["imu_temperature_C"], places=5)
-        self.assertAlmostEqual(rssi_in, unpacked["RSSI_dB"], places=5)
-        self.assertAlmostEqual(fei_in, unpacked["FEI_Hz"], places=5)
-        testing.assert_array_almost_equal(gyro_in, unpacked["gyro"])
-        testing.assert_array_almost_equal(mag_in, unpacked["mag"])
+        self.assertEqual(state_in, unpacked.state_index)
+        self.assertEqual(f_contact_in, unpacked.contact_flag)
+        self.assertEqual(f_burn_in, unpacked.burn_flag)
+        self.assertEqual(error_count_in, unpacked.software_error_count)
+        self.assertEqual(boot_count_in, unpacked.boot_count)
+        self.assertAlmostEqual(vbatt_in + 0.01, unpacked.battery_voltage, places=5)
+        self.assertAlmostEqual(cpu_temp_in, unpacked.cpu_temperature_C, places=5)
+        self.assertAlmostEqual(imu_temp_in, unpacked.imu_temperature_C, places=5)
+        self.assertAlmostEqual(rssi_in, unpacked.RSSI_dB, places=5)
+        self.assertAlmostEqual(fei_in, unpacked.FEI_Hz, places=5)
+        testing.assert_array_almost_equal(gyro_in, unpacked.gyro)
+        testing.assert_array_almost_equal(mag_in, unpacked.mag)
```

Comment:
Add unit tests for the entire `telemetry_packet` here.

Reply:
Thanks! Updated with a test for each value in the telemetry packet